<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerService</class>
 <widget class="QDialog" name="CustomerService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 223, 223);</string>
  </property>
  <widget class="QScrollArea" name="ContactUs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>479</width>
      <height>429</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_GetInTouch">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="text">
      <string>Get in touch with us </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFirstName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FirstNameBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CompanyNameBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LastNameBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Messagebox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>370</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 145, 145);</string>
     </property>
     <property name="text">
      <string>SEND</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCompanyName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Company Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 185, 185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBackMain">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 145, 145);</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
